models/preles/R/runPRELES.jobsh.R

```diff
@@ -15,21 +15,19 @@
 ##' @param end_date End time of the simulation
 ##' @export 
 ##' @author Tony Gardella, Michael Dietze
-runPRELES.jobsh<- function(met.file,outdir,start.date,end.date){
+runPRELES.jobsh<- function(met.file,outdir,priors,start.date,end.date){
```

Comment:
new argument should be parameters not priors -- you're always passing a specific set of parameter values, not a PDF, and most often those parameters are coming from the posterior not  the prior.